Release_Notes.md

```diff
@@ -1,6 +1,14 @@
 # FLEDGE Release Notes
 
 
+## Chrome M107
+
+
+
+*   Bug fixes:
+    *   ['forDebuggingOnly' URLs only replaces post-auction signals placeholders in query parameters now, instead of path](http://crbug.com/1338233).
```

Comment:
```suggestion
    *   [The `forDebuggingOnly.reportAdAuctionLoss()` API was fixed to perform post-auction signal parameter replacement on URL query parameters instead of the URL path](http://crbug.com/1338233).
```

Reply:
Done.
I think worth calling out both `forDebuggingOnly.reportAdAuctionLoss()` and `forDebuggingOnly.reportAdAuctionWin()` APIs?